Xamarin.Forms.Platform.Android.AppLinks/AndroidAppLinks.cs

```diff
@@ -35,19 +35,21 @@ public AndroidAppLinks(Context context)
 			_client.Connect();
 		}
 
-		public void DeregisterLink(IAppLinkEntry appLink)
+		public Task DeregisterLink(IAppLinkEntry appLink)
 		{
 			RemoveFromIndexItemAsync(appLink.AppLinkUri.ToString());
+			return Task.FromResult<object>(null);
```

Comment:
If there is an existing guideline or practice how this should be handled in the Forms repo please let me know and I'll change it.